src/Splitio-net-core/Services/Client/Classes/SelfRefreshingClient.cs

```diff
@@ -130,11 +129,20 @@ private void ReadConfig(ConfigurationOptions config)
             EventsBaseUrl = string.IsNullOrEmpty(config.EventsEndpoint) ? "https://events.split.io" : config.EventsEndpoint;
             SplitsRefreshRate = config.FeaturesRefreshRate ?? 5;
             SegmentRefreshRate = config.SegmentsRefreshRate ?? 60;
-            HttpEncoding = "gzip";
             HttpConnectionTimeout = config.ConnectionTimeout ?? 15000;
             HttpReadTimeout = config.ReadTimeout ?? 15000;
+
+#if NETSTANDARD
```

Comment:
You use the same snippet at the top, is it possible to use some utils library to get `SdkVersion` and `SdkSpecVersion` to avoid repeating code?